### ISSUE TITLE
MDAnalysis

### ISSUE THREAD

**Issue by noorehh** [ubccr/software-layer#232]
I didn't see MDAnalysis available in the ccrsoft/2023.01 module, can you please help me install it?

https://github.com/MDAnalysis/mdanalysis

Thank you,

**Comment by peter-zhang-chem** [comment]
Dear CCR team,
Here I attached the easybuild recipe for MDAnalysis 2.4.2.
https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/MDAnalysis

Thank you!

**Comment by dsajdak** [comment]
@peter-zhang-chem @noorehh this module has been published to the ccrsoft/2023.01 software repository.  You can find it and load it like this:

```
vortex:~$ module spider analysis

----------------------------------------------------------------------------------------------------------------
  MDAnalysis: MDAnalysis/2.4.2 (E)
----------------------------------------------------------------------------------------------------------------
    This extension is provided by the following modules. To access the extension you must load one of the following modules. Note that any module names in parentheses show the module location in the software hierarchy.


       mdanalysis/2.4.2 (gcc/11.2.0 openmpi/4.1.1)


Names marked by a trailing (E) are extensions provided by another module.



----------------------------------------------------------------------------------------------------------------
  mdanalysis: mdanalysis/2.4.2
----------------------------------------------------------------------------------------------------------------
    Description:
      MDAnalysis is an object-oriented Python library to analyze trajectories from molecular dynamics (MD)
      simulations in many popular formats.


    You will need to load all module(s) on any one of the lines below before the "mdanalysis/2.4.2" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    This module provides the following extensions:

       biopython/1.81 (E), fasteners/0.18 (E), GridDataFormats/1.0.1 (E), gsd/2.8.0 (E), MDAnalysis/2.4.2 (E), mmtf-python/1.1.3 (E), mrcfile/1.4.3 (E), msgpack/1.0.5 (E)

vortex:~$ module load gcc/11.2.0  openmpi/4.1.1 mdanalysis/2.4.2

```
